css/style.css

```diff
@@ -1,15 +1,458 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	overflow-x: hidden;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_Axion/pullRequest?prid=5457135)